### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by Soline324**
@gingerkidney PTAL~

### CODE REVIEW

**Comment by gingerkidney** [comment]
LGTM

File: incubator/README.md [hunk 1]
```diff
@@ -232,7 +232,7 @@ get full support by PingCAP and the community.
   - PingCAP
     - Official learning resources
     - Necessary tutoring lessons
-    - Marketing campaign and promotion  for the project and
+    - Marketing campaign and promotion  for the project
```
Comment (lilin90) [inline incubator/README.md:235]:
```suggestion
    - Marketing campaign and promotion for the project
```

**Comment by lilin90** [comment]
@Soline324 Please sign the CLA.

**Comment by Soline324** [comment]
> @Soline324 Please sign the CLA.

Done~

**Review by winkyao — APPROVED** [review]

**Pull request merged**